src/apps/frontend/pages/authentication/login/login-form.tsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
```

Comment:
We are disabling this in multiple files. What is about and can we look into addressing it?